bom/pom.xml

```diff
@@ -103,11 +105,12 @@
                 <groupId>org.hibernate</groupId>
                 <artifactId>hibernate-envers</artifactId>
                 <version>${hibernateVersion}</version>
-            </dependency>
```

Comment:
I removed it because we don't use it anywhere? Is there any particular reason to keep it?

Anyway, I kept the change in a separate commit because I wasn't sure about it